pkg/lib/config/captcha.go

```diff
@@ -5,24 +5,24 @@ var _ = Schema.Add("CaptchaConfig", `
 	"type": "object",
 	"additionalProperties": false,
 	"properties": {
-		"provider": { "$ref": "#/$defs/CaptchaProvider" }
+		"provider": { "$ref": "#/$defs/LegacyCaptchaProvider" }
 	}
 }
 `)
 
 type CaptchaConfig struct {
-	Provider *CaptchaProvider `json:"provider,omitempty"`
+	LegacyProvider *LegacyCaptchaProvider `json:"provider,omitempty"`
```

Comment:
- Rename the field LegacyProvider to Deprecated_Provider
- Rename the type LegacyCaptchaProvider to Deprecated_CaptchaProvider